src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -110,7 +111,12 @@ public async Task<ServerNode> GetCurrentSessionNode()
         /// Translate between an ID and its associated entity
         /// </summary>
         internal readonly Dictionary<string, DocumentInfo> IncludedDocumentsById = new Dictionary<string, DocumentInfo>(StringComparer.OrdinalIgnoreCase);
-
+        
+        /// <summary>
+        /// Translate between an CV and its associated entity
+        /// </summary>
+        internal readonly Dictionary<string, DocumentInfo> IncludedRevisionByChangeVectors = new Dictionary<string, DocumentInfo>(StringComparer.OrdinalIgnoreCase);
```

Comment:
This will almost always be null, let's initialize this only when it is actually used.